### PR TITLE
[type:feat]Add namespace management interface and plugin management interface adaptation and transformation

### PR DESCRIPTION
**Pull request by xcsnx**

### CODE REVIEW

**Comment by kerwin612** [comment]
Awesome job, really well done!



a93bef91d67006aec6a2f5ad4e9d21a


However, could you attach some design details?
I've tested your changes, but I have a few questions:

1. What's the difference between **Plugin** and **PluginTemplate**? To me, the two pages seem almost 90% identical, which might confuse our users.
2.  I traced the requests and noticed that the **Plugin** page seems to have some association with **Namespace**, but I can't find any indication of this relationship on the interface.
3.  I switch to the **Namespace** page, added a new record, refreshed the page, and switched to the **Plugin** page, but I didn't notice any difference. Why is that? If this feature is not implemented at this stage, it would be better to have a corresponding prompt on the interface.
***I'm putting myself in the shoes of someone upgrading from the current version to yours, and these are the issues I encountered. I hope you can address my concerns. ***

**Review by kerwin612 — CHANGES_REQUESTED** [review]
Could you attach some design details?

**Comment by xcsnx** [comment]
> Could you attach some design details?

Thank you for your comments. Regarding the first question, a plugin is an instance of a plugin template under a specific namespace. Modifications to the plugin configuration under a particular namespace do not alter the plugin template configuration. As for the second and third questions, the general component for switching namespaces on the Plugin interface has not been developed yet, so related associations cannot be found. As you mentioned, indeed, certain functionalities have not been implemented in the current stage. I will consider adding corresponding prompts to the interface. Lastly, you can find more useful information in the namespace transformation document at: https://summer-ospp.ac.cn/previewPdf/201

File: src/common/router.js [hunk 1]
```diff
@@ -144,6 +144,14 @@ export const getRouterData = (app) => {
         () => import("../routes/System/Alert"),
       ),
     },
+    "/system/namespace": {
+      // 告警管理
```
Comment (kerwin612) [inline src/common/router.js:148]:
```suggestion
      // 命名空间管理
```

**Review by kerwin612 — APPROVED** [review]

File: src/common/router.js [hunk 2]
```diff
@@ -144,6 +144,14 @@ export const getRouterData = (app) => {
         () => import("../routes/System/Alert"),
       ),
     },
+    "/config/namespace": {
+      // 命名空间管理
```
Comment (dengliming) [inline src/common/router.js:148]: Hey, We suggest using English comment. : )

**Review by dengliming — APPROVED** [review]

**Pull request merged**